robot/src/main/java/us/ilite/robot/auto/paths/FieldElementLocations.java

```diff
@@ -23,7 +23,7 @@
     //-----------------
     // Loading Station
     //-----------------
-    public static final Translation2d kLoadingStation = new Translation2d();
+    public static final Translation2d kLoadingStation = new Translation2d(0.0, 2.0 * 12.0);
```

Comment:
What is it 2.0 * 12, why not just hardcode to 24.0?

Reply:
I was trying to be consistent - nearly all of my field measurements are converted from feet -> inches